backend/sozisel/lib/sozisel_web/schema/resolvers/session_recording_resolvers.ex

```diff
@@ -0,0 +1,205 @@
+defmodule SoziselWeb.Schema.Resolvers.SessionRecordingResolvers do
+  alias SoziselWeb.Context
+  alias Sozisel.Repo
+  alias SoziselWeb.Schema.Helpers
+  alias SoziselWeb.Schema.Subscriptions.Topics
+  alias Sozisel.Model.Sessions
+  alias Sozisel.Model.Sessions.Session
+  alias Sozisel.Model.SessionRecordings
+  alias Sozisel.Model.SessionRecordings.SessionRecording
+  alias Sozisel.Model.Polls.PollResult
+
+  import SoziselWeb.Schema.Middleware.ResourceAuthorization, only: [fetch_resource!: 2]
+
+  require Logger
+
+  @media_storage_module Sozisel.MediaStorage.Disk
+
+  def get_session(_parent, %{id: session_id}, ctx) do
+    user_id = Context.current_user!(ctx).id
+
+    with %Session{user_id: ^user_id} = session <- Repo.get(Session, session_id) do
+      {:ok, session}
+    else
+      %Session{} -> {:error, :unauthorized}
+      nil -> {:ok, nil}
+    end
+  end
+
+  def get_session_thumbnail(_parent, %{id: session_id}, _ctx) do
+    with %Session{} = session <- Repo.get(Session, session_id) do
+      session =
+        session
+        |> Repo.preload(session_template: [:agenda_entries])
+
+      thumbnail =
+        session
+        |> Map.put(:password_required, session.entry_password != nil)
+        |> Map.put(:agenda_entries, session.session_template.agenda_entries)
+        |> Map.put(
+          :estimated_time,
+          session.session_template.estimated_time
+        )
+
+      {:ok, thumbnail}
+    else
+      nil -> {:ok, nil}
+    end
+  end
+
+  def get_session_recording(_parent, %{id: id}, _ctx) do
```

Comment:
Why did you name the file `session_recording_resolvers.ex` and then put all session related stuff here instead of keeping it in the original file?